.github/workflows/deploy-toss-backend.yml

```
@@ -33,10 +33,33 @@ jobs:
+          mkdir -p ~/deploy/certs
+          printf '%s' "$PRIVATE_KEY" | base64 --decode > ~/deploy/certs/davinci_private.key 
+          printf '%s' "$PUBLIC_CERT" | base64 --decode > ~/deploy/certs/davinci_public.cert
+
+          chmod 600 ~/deploy/certs/davinci_private.key 
+          chmod 644 ~/deploy/certs/davinci_public.cert
+        env:
+          PRIVATE_KEY: ${{ secrets.DAVINCI_PRIVATE_KEY }}
+          PUBLIC_CERT: ${{ secrets.DAVINCI_PUBLIC_CERT }}
+
+      - name: Build image
+        working_directory: ./deploy-toss
```

actionlint (1.7.12)

[error] unexpected key "working_directory" for step to run shell command. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "working-directory"

(syntax-check)